tests/h/services/organization_test.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pytest
+
+from h.models import Organization
+from h.services.organization import OrganizationService
+from h.services.organization import organization_factory
+
+
+class TestOrganizationService(object):
+    def test_create_organization_returns_organization(self, service):
```

Comment:
These test names still reference the previous method name (`test_create_organization*`)